docs/src/Theory/Common/SurfaceFluxes.md

```diff
@@ -1,105 +1,283 @@
 
-```@example byun1990
-plot(Ri_range,
-     [Byun1990.compute_exchange_coefficients(param_set, Ri, scale * z_0, z_0,
-                                             γ_m, γ_h, β_m, β_h, Pr_0)[2]
-        for Ri in Ri_range, scale in scales],
-     xlabel = "Bulk Richardson number (Ri_b)",
-     ylabel = "Exchange coefficient",
-     title = "Heat exchange coefficient",
-     labels = scales, legendtitle=L"z/z_0")
+In each of these cases $\hat{n}$ corresponds to the upward pointing unit-normal vector. 
 
-savefig("exchange_byun1990_fig4b.svg") # hide
-nothing # hide
+## Equations
+
+We start by defining the universal functions for the velocity, enthalpy and moisture terms. _NK_ express the Monin-Obukhov length in terms of the potential temperature 
+
+```math 
+L_{MO} = -\frac{u_{\star}^3 \overline{\theta}}{\kappa g \overline{w^{\prime}\theta^{\prime}}|_{sfc}} 
 ```
-![](exchange_byun1990_fig4b.svg)
 
-Recreation of Figure 4(b) from Byun (1990)
+From the above definitions for the fluxes in the moist, compressible flow equations, we substitute the potential temperature flux $\overline{w^{\prime}\theta^{\prime}} = J/(\rho c_{pm})$, $\overline{w^{\prime}u^{\prime}} = \tau/\rho)$ to recover eq (1) in _GLGS_. 
 
-## `Nishizawa2018`
+```math 
+L_{MO} = -\frac{(\tau/\rho)^{(3/2)} \overline{\theta}}{\kappa g \frac{J}{(\rho c_{pm})}|_{sfc}} 
+```
 
-### Plots
+Given the formulation of the compressible flow equations in `ClimateMachine`  we arrive at the following formulation for the Monin-Obukhov similarity length
 
-```@example nishizawa2018
-using ClimateMachine.SurfaceFluxes.Nishizawa2018
-using Plots, LaTeXStrings
-using CLIMAParameters
-struct EarthParameterSet <: AbstractEarthParameterSet end
-param_set = EarthParameterSet()
+```math 
+L_{MO} = - \frac{\overline{\theta} u_{\star}^3}{\kappa g [\overline{w^{\prime}\theta^{\prime}} + \overline{w^{\prime}q_{t}^{\prime}}]} = - \frac{\overline{\theta} u_{\star}^3}{\kappa g [\frac{J}{\rho c_{pm}} + \theta\frac{D}{\rho L_{v}}]}
+```
 
-FT = Float64
+NK use volume-averaged quantities$\langle X \rangle$ such that 
+```math 
+\langle X \rangle = \frac{1}{\Delta z} \int_{0}^{\Delta z} X(z) dz
+```
 
-a = FT(4.7)
-θ = FT(350)
-z_0 = FT(10)
-u_ave = FT(10)
-flux = FT(1)
-Δz = range(FT(10.0), stop=FT(100.0), length=100)
-Ψ_m_tol, tol_abs, iter_max = FT(1e-3), FT(1e-3), 10
+In the `ClimateMachine` DG discretization we interpret these as the nodal variable values at a given model reference altitude $\Delta z$. This value is currently taken as the first interior grid point in the model column description, which typically is of $O(1-10)$ in LES configurations and $O(10-100)$ in the GCM configurations. An alternative to this is obtained by fixing the reference $\Delta z$ and interpolating the DG polynomials to determine variable values at the specified model height. Applying _NK_ equations (17), (18) to the `ClimateMachine` equations we arrive at 
```

Comment:
I'm confused by this statement. In NK, the use of volume-averaged quantities leads to modified similarity functions. If we use nodal values (interpreted as point values), the NK approach does not apply, and we should use other standard similarity functions (e.g., those in GLGS). It would be good to clarify what we are doing.